source/codegen/metadata/nirfmxspecan_restricted/config.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+config = {
+    "code_readiness": "RestrictedNextRelease",
+    "api_version": "21.0.0",
+    "restricted_driver_service": True,
```

Comment:
I think you can remove this `"restricted_driver_service": True,` now and update your new `is_restricted_driver_service` helper to look at `code_readiness` instead.

Reply:
Fixed in commit [e63fe2a](https://github.com/ni/grpc-device/pull/689/commits/e63fe2a19661c5774909bfb32d2f2d940c609cbe).